manifests/overlays/rhoai/kustomization.yaml

```diff
@@ -0,0 +1,86 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+commonLabels:
+  app: rhods-dashboard
+  app.kubernetes.io/part-of: rhods-dashboard
+bases:
```

Comment:
btw @zdtsw @VaishnaviHire we kept bases here cause it was giving issues in v1 operator, can we change it to `resources already?`

Reply:
we have not updated kustomize to v5, but i think `resources` should work, can try that